### DISCUSSION TITLE
convert deficits w.r.t. tiff2pdf

### DISCUSSION THREAD

**Discussion by computerix** [ImageMagick/ImageMagick#6668] [Contribute]
### Is your feature request related to a problem? Please describe.

The libtiff team no longer supports tiff2pdf, it will most likely be removed completely soon.
Everyone suggests using imagemagick convert instead of tiff2pdf to convert tiff to pdf, 
but currently, convert is by far inferior to tiff2pdf:

* At least for monochrome tiff's and zip compression (which is the most important case here), the file size of the pdf produced by convert is several times larger than the pdf produced by tiff2pdf for the same input. 
This is a showstopper here.
* convert is slower by orders of magnitude (again, tested with monochrome tiff's and zip compression). Many-page batch scans are the most common use case here, so this is a problem.

Example: 8 page high-res monochrome text tiff: 0.064 sec versus 1.713 sec conversion time, 537486 versus 1444042 bytes pdf file size.

Moreover, convert lacks the options tiff2pdf offers to set the atributes in the generated pdf file (author, creator, date, title, subject, keywords, ...).

### Describe the solution you'd like

To replace tiff2pdf, imagemagick convert should

* produce output files as small as those produced by tiff2pdf
* not take significantly longer than tiff2pdf
* offer all options and features tiff2pdf currently offers.

**Comment by urban-warrior** [comment]
Rather than transforming ImageMagick into a direct substitute for the tiff2pdf tool, we suggest that you establish a GitHub project and persist in upholding and enhancing tiff2pdf.
